src/decision/CriteriaWeights.jl

```diff
@@ -1,180 +1,172 @@
         name="Low Coral Cover",
         description="Higher values give greater preference to sites with low coral cover for seeding deployments.",
     )
-    coral_cover_high::P = Param(
+    coral_cover_high::U2 = Factor(
         0.0;
-        ptype="real",
-        bounds=(0.0, 1.0),
-        default_bounds=(0.0, 1.0),
-        dists="unif",
+        ptype="continuous",
+        dist=Uniform,
+        dist_params=(0.0, 1.0),
         criteria_keywords=(:fog, :weight),
         name="High Coral Cover",
         description="Higher values give preference to sites with high coral cover for shading deployments.",
     )
-    seed_priority::P = Param(
+    seed_priority::U2 = Factor(
         1.0;
-        ptype="real",
-        bounds=(0.0, 1.0),
-        default_bounds=(0.0, 1.0),
-        dists="unif",
+        ptype="continuous",
+        dist=Uniform,
+        dist_params=(0.0, 1.0),
         criteria_keywords=(:seed, :weight),
         name="Predecessor Priority (Seed)",
         description="Importance of seeding sites that provide larvae to priority reefs.",
     )
-    fog_priority::P = Param(
+    fog_priority::U2 = Factor(
         0.0;
-        ptype="real",
-        bounds=(0.0, 1.0),
-        default_bounds=(0.0, 1.0),
-        dists="unif",
+        ptype="continuous",
+        dist=Uniform,
+        dist_params=(0.0, 1.0),
         criteria_keywords=(:fog, :weight),
         name="Predecessor Priority (Shade)",
         description="Importance of shading sites that provide larvae to priority reefs.",
     )
-    seed_zone::P = Param(
+    seed_zone::U2 = Factor(
         0.0;
-        ptype="real",
-        bounds=(0.0, 1.0),
-        default_bounds=(0.0, 1.0),
-        dists="unif",
+        ptype="continuous",
+        dist=Uniform,
+        dist_params=(0.0, 1.0),
         criteria_keywords=(:seed, :weight),
         name="Zone Predecessor (Seed)",
         description="Importance of seeding sites that provide larvae to priority (target) zones.",
     )
-    fog_zone::P = Param(
+    fog_zone::U2 = Factor(
         0.0;
-        ptype="real",
-        bounds=(0.0, 1.0),
-        default_bounds=(0.0, 1.0),
-        dists="unif",
+        ptype="continuous",
+        dist=Uniform,
+        dist_params=(0.0, 1.0),
         criteria_keywords=(:fog, :weight),
         name="Zone Predecessor (Shade)",
         description="Importance of shading sites that provide larvae to priority (target) zones.",
     )
-    deployed_coral_risk_tol::P2 = Param(
+    deployed_coral_risk_tol::U3 = Factor(
         1.0;
-        ptype="real",
-        bounds=(0.75, 1.0),
-        default_bounds=(0.0, 1.0),
-        dists="unif",
+        ptype="continuous",
+        dist=Uniform,
+        dist_params=(0.0, 1.0),
         criteria_keywords=(:threshold, :seed, :fog),
         name="Risk Tolerance",
         description="Filters out sites with heat/wave stress above threshold.",
     )
-    depth_min::P3 = Param(
+    depth_min::DO = Param(
```

Comment:
@ConnectedSystems Why haven't you changed this one to `Factor`?

Reply:
Ooops